Framework/Core/test/test_LogParsingHelpers.cxx

```diff
@@ -0,0 +1,29 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#define BOOST_TEST_MODULE Test Framework LogParsingHelpers
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_DYN_LINK
+
+#include "Framework/LogParsingHelpers.h"
+#include <boost/test/unit_test.hpp>
+#include <regex>
```

Comment:
remnant from testing?

Reply:
No, not really, I do want to make sure I have a test which fails if the log format changes and I adapt `parseTokenLevel`.